CMakeLists.txt

```diff
@@ -227,9 +227,44 @@ include(Compilers)
 message(STATUS "Configuring build settings...")
 include(BuildSettings)
 
+if (MSVC)
+    # TODO: These msvc specific settings probably belong in BuildSettings. 
+    # But I am not sure where to put them, so I will leave them in the obvious
+    # place for now.
+
+    # set debug postfix so debug library and executable artifact names do not
+    # conflict with release artifacts
+    set(CMAKE_DEBUG_POSTFIX "_d")
+
+    # VTune is much more helpful when debug information is included in the 
+    # generated release code. 
+    add_definitions(/Zi)
```

Comment:
this should probably be `target_compile_definitions(timemory-compile-options INTERFACE /Zi)`